students/cbrown/Lesson 9/mailroom_oo_final.py

```diff
@@ -0,0 +1,224 @@
+import sys
+
+
+class Donor_Collection():
+
+    donor_dict = {"Bill Gates": [539000, 235642],
+          "Jeff Bezos": [108356, 204295, 897345],
+          "Satya Nadella": [236000, 305352],
+          "Mark Zuckerberg": [153956.35],
+          "Mark Cuban":[459035, 369.50, 570.89]}
```

Comment:
donor_dict should not be a class attribute but instead should be an instance attribute (defined in `__init__`)